cinema_service/settings.py

```diff
@@ -21,7 +26,8 @@
 
 # SECURITY WARNING: keep the secret key used in production secret!
 SECRET_KEY = (
-    "django-insecure-6vubhk2$++agnctay_4pxy_8cq)mosmn(*-#2b^v4cgsh-^!i3"
+    "django-insecure-6vubhk2$++agnctay_4pxy_8cq)"
+    "mosmn(*-#2b^v4cgsh-^!i3"
```

Comment:
It's not recommended to hardcode the `SECRET_KEY` in the settings file. Consider using an environment variable to securely manage the secret key.